ListToArray.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Arrays;
+import java.util.List;
+
+class InputOutputValue {
+    String[] str;
```

Comment:
why cann't we pass the str around the methods instead of having a instance variable?